ci-operator/step-registry/ipi/conf/nutanix/multi-nics/ipi-conf-nutanix-multi-nics-commands.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+if [[ -f ${CLUSTER_PROFILE_DIR}/secrets.sh ]]; then
+  NUTANIX_AUTH_PATH=${CLUSTER_PROFILE_DIR}/secrets.sh
+else
+  NUTANIX_AUTH_PATH=/var/run/vault/nutanix/secrets.sh
```

Comment:
Seem like missing the following part in ref yaml:
```
  credentials:
  - namespace: test-credentials
    name: nutanix-pc
    mount_path: /var/run/vault/nutanix
```

Reply:
It's added in context [step](https://github.com/openshift/release/blob/1702fa8d7b90cbac5204c44b36c78e51aefef554/ci-operator/step-registry/ipi/conf/nutanix/context/ipi-conf-nutanix-context-ref.yaml#L9) , do we need to add it here?

Reply:
That is for `ipi-conf-nutanix-context` step, but in this step, this line also reference the path in the code, so need the setting, right ?

Reply:
ok, updated the code